geometry_msgs/Point32 offset
float32 tof
float32 range
